HeterogeneousCore/CUDAServices/src/CUDAService.cc

```diff
@@ -36,6 +36,12 @@ CUDAService::CUDAService(edm::ParameterSet const& iConfig, edm::ActivityRegistry
     return;
   }
 
+  auto numberOfStreamsPerDevice = iConfig.getUntrackedParameter<unsigned int>("numberOfStreamsPerDevice");
+  if(numberOfStreamsPerDevice > 0) {
+    numberOfStreamsTotal_ = numberOfStreamsPerDevice * numberOfDevices_;
+    edm::LogWarning("CUDAService") << "Number of edm::Streams per CUDA device has been set to " << numberOfStreamsPerDevice << ". With " << numberOfDevices_ << " CUDA devices, this means total of " << numberOfStreamsTotal_ << " edm::Streams for all CUDA devices.";
```

Comment:
LogWarning instead of LogInfo because LogWarning gets printed by default. I'd eventually silence it (to LogDebug), but for the beginning I think a printout of reminding what it does could be useful. Also the core framework itself prints the number of streams and threads...

Reply:
Then I'd go with LogSystem or something like that for the time being (same level as LogError but un-silenceable IIRC)

Reply:
Ok. I'll also add a comment to eventually silence it.

Reply:
Done (squashed to last commit).